docs/source/examples/combine_ddd_zne.md

```diff
@@ -0,0 +1,241 @@
+```
+
+Let's see what the execute function returns when we call it on our GHZ circuit, just leaving the default noise levels:
+
+```{code-cell}
+res = execute(circuit)
+res.to_dict() # Dictionary for more convenient visualization
+```
+
+The `'counts'` key represents the number of times a particular measurement bitstring occurred. 
+
+Since the default execute function includes noise in this case, we see that most of the results are all ones or all zeros, corresponding to perfect entanglement between all qubits.
+
+A few others, like `'110111'` however are one or more bitflips away from either of these states. It is these types of errors we want to mitigate. 
+
++++
+
+## Observable
+
+In this example, we just want to check if we have achieved entanglement across all qubits. In this case, we will measure the observable $⨂_{i=1}^n​X_i$ or measuring `X` on all qubits. 
+
+This corresponds to projecting the state onto either the $∣+⟩^{⊗n}$ or $∣−⟩^{⊗n}$ basis. For a perfect GHZ state, this observable will have an expectation value of 1, which corresponds to all qubits being in the same state.
+
+```{code-cell}
+obs = Observable(PauliString("X" * num_qubits))
+print(obs)
+```
+
+For the circuit defined above, the ideal (noiseless) expectation value of the observable is 1:
+
+```{code-cell}
+from functools import partial
+
+ideal_exec = partial(execute, rz_noise = 0.0, depolar_noise = 0.0)
+
+ideal = obs.expectation(circuit, ideal_exec)
+print("Ideal value:", "{:.5f}".format(ideal.real))
+```
+
+The unmitigated (noisy) result however, is impacted by the `Rz` and depolarizing errors. In fact, we are not even halfway to the correct expectation value:
+
+```{code-cell}
+noisy_exec = partial(execute, rz_noise = 0.02, depolar_noise = 0.005)
+noisy = obs.expectation(circuit, noisy_exec) 
+print("Unmitigated noisy value:", "{:.5f}".format(noisy.real))
+```
+
+### Applying Digital Dynamical Decoupling
+
+Next we choose our gate sequences to be used in the digital dynamical decoupling routine (DDD). 
+More information on choosing appropriate sequences can be found in the [DDD theory](../guide/ddd-5-theory.md#common-examples-of-ddd-sequences) section of the user guide.
+
+To do this, we will insert DDD sequences into our circuit itself and then compare the original circuit with the modified one.
+
+```{code-cell}
+from mitiq.ddd import insert_ddd_sequences, rules
+
+print("Original circuit \n", circuit)
+
+rule = rules.yy
+
+ddd_circuit = insert_ddd_sequences(circuit, rule)
+print("DDD modified circuit \n", ddd_circuit)
+```
+
+Now we execute our function on the `ddd_circuit`:
+
+```{code-cell}
+ddd_noisy = obs.expectation(ddd_circuit, noisy_exec)
+
+print("Unmitigated expectation value:", "{:.5f}".format(noisy.real))
+
+print("Expectation value with DDD:", "{:.5f}".format(ddd_noisy.real))
+```
+
+### Zero Noise Extrapolation alone
+
++++
+
+For comparison, we then apply ZNE one our original circuit (without DDD sequences inserted).
```

Comment:
s/one/on

Reply:
?

Reply:
I was just pointing out the typo. You wanted `on` instead of `one`. I was using the sed notation `s/match/replacement`, unix slang :D